cosmo_tester/test_suites/service_composition/shared_resource_test.py

```diff
@@ -20,3 +24,70 @@ def test_shared_resource(image_based_manager, ssh_key, logger, test_config):
         _verify_deployments_and_nodes(app, 2)
         _check_custom_execute_operation(app, logger)
         _verify_custom_execution_cancel_and_resume(app, logger)
+
+
+@pytest.fixture(scope='function')
+def managers(request, ssh_key, module_tmpdir, test_config, logger):
+    hosts = Hosts(ssh_key, module_tmpdir, test_config, logger, request, 2)
+    passed = True
+    try:
+        hosts.create()
+        yield hosts.instances
+    except Exception:
+        passed = False
+        raise
+    finally:
+        hosts.destroy(passed=passed)
+
+
+def test_external_shared_resource_idd(managers, ssh_key, logger, test_config):
+    tenant = 'test_external_shared_resource_idd'
+    local_mgr, external_mgr = managers
+
+    ext_username = 'dave'
+    ext_password = 'swordfish'
+    external_mgr.run_command('cfy users create -r sys_admin -p {0} {1}'.format(
```

Comment:
This should probably be external_mgr.client.users.create(ext_username, ext_password, 'sys_admin') - best to avoid using the CLI where we don't need to.